GitUI/LeftPanel/RepoObjectsTree.cs

```diff
@@ -694,7 +703,7 @@ public void OpenContextMenu()
 
                 foreach (string text in nodeTexts)
                 {
-                    node = nodes.SingleOrDefault(node => node.Text == text);
+                    node = nodes.SingleOrDefault(n => n.Text == text);
```

Comment:
There is no need to abbreviate.

Reply:
there is a warning that `node` in this place hides a variable of the same name from the outer scope.